src/Discord.Net.WebSocket/Entities/Guilds/SocketGuild.cs

```diff
@@ -337,6 +337,15 @@ public IReadOnlyCollection<SocketThreadChannel> ThreadChannels
         public IReadOnlyCollection<SocketForumChannel> ForumChannels
             => Channels.OfType<SocketForumChannel>().ToImmutableArray();
```

Comment:
Might want to add a check to not include media channels here

Reply:
This way it matches the behavior of similar methods.
ex: `SocketGuild.TextChannels`, `SocketGuild.VoiceChannels`